components/EditOutivity/index.js

```diff
@@ -0,0 +1,156 @@
+import styled from "styled-components";
+import { useState } from "react";
+
+export default function EditOutivityForm({
+  onEditOutivity,
+  outivity,
+  setIsEdit,
```

Comment:
Since you use this form only for editing I am wondering about the purpose of ``setIsEdit``.

Reply:
Ah I see: You use it for condiitional rendering the detailspage / the form. It is okay so.